src/GroebnerBasis.jl

```diff
@@ -49,7 +46,7 @@ function __init__()
 end
 
 function versioninfo()
-    print("GroebnerBasis v0.1.0\n")
+    print("GroebnerBasis v0.2.0\n")
```

Comment:
It would be better to print the actual version here; we have a pkgversion() function for this in Oscar.jl